packages/grafana-data/src/transformations/transformers/merge.test.ts

```diff
@@ -196,6 +196,127 @@ describe('Merge multipe to single', () => {
     expect(unwrap(result[0].fields)).toEqual(expected);
   });
 
+  it('combine two tables, where first is partial overlapping, into one', () => {
+    const cfg: DataTransformerConfig<MergeTransformerOptions> = {
```

Comment:
Just a nit, maybe move this inside the describe scope to avoid duplication?

Reply:
Awesome, feedback! I will update. Thanks 🙏